cli.py

```diff
@@ -18,6 +18,16 @@
 
 app = create_app(run_as_server=False)
 
+def create_su():
+    '''Create the first super user'''
+    with app.app_context():
+        from pybossa.core import user_repo
+        from pybossa.model.user import User
+        if not user_repo.get_all():
+            user = User(email_addr='user@user.com', name='user', fullname='user', admin=True)
+            user.set_password('test')
```

Comment:
I am wondering if we should consider omitting this PR due to exposing the account credentials openly. We could instead opt to allow local installs to follow documentation for creating the accounts. Thoughts?

Reply:
Yes, I think that is a good suggestion. I was considering this change in the context of running a local dev environment, but this function is agnostic to that aspect and users may not adopt best practices and change the credentials in production.

Reply:
I just had a thought: we could also prompt the user for info like so:
```
fullname = input("Enter your full name: ")
email = input("Enter your email address: ")
name = input("Choose a username: ")
password = input("Choose a password: ")
```

Reply:
This could be reasonable Can we ensure that `create_su()` is only called manually by dev?

Reply:
+1 to Kory's suggestion

Reply:
Considering valid security concerns raised by Kory/Noor and since its only for local instance, this could be part of internal documentation rather than code.